tests/function-name/default/test.ts.lint

```diff
@@ -0,0 +1,88 @@
+function fooGood() {}
+class MyClassGood {
+    foo() {}
+    foo1() {}
+    myFoo() {}
+    myFoo1() {}
+
+    private bar() {}
+    private bar1() {}
+    private myBar() {}
+    private myBar1() {}
+}
+
+class MyClassGoodStatic {
+    static FOO() {}
+    static FOO_BAR() {}
+    static FOO_2() {}
+    static FO__O() {}
```

Comment:
No test for `static _FOO() {}`?
Should be allowed by RegExp, but can't find in success/failed cases.

Reply:
They don't exist in the original tests so I didn't copy them here. I can add them in now.